flink-sql-connector-kinesis/pom.xml

```diff
@@ -67,21 +67,18 @@ under the License.
                         <configuration>
                             <artifactSet>
                                 <includes>
-                                    <include>org.apache.flink:flink-connector-base</include>
-                                    <include>org.apache.flink:flink-connector-kinesis</include>
-                                    <include>com.fasterxml.jackson.core:jackson-core</include>
-                                    <include>com.fasterxml.jackson.core:jackson-databind</include>
-                                    <include>com.fasterxml.jackson.core:jackson-annotations</include>
-                                    <include>com.fasterxml.jackson.dataformat:jackson-dataformat-cbor</include>
-                                    <include>joda-time:joda-time</include>
-                                    <include>commons-codec:commons-codec</include>
-                                    <include>commons-io:commons-io</include>
-                                    <include>commons-lang:commons-lang</include>
-                                    <include>commons-logging:commons-logging</include>
-                                    <include>org.apache.commons:commons-lang3</include>
-                                    <include>com.google.guava:guava</include>
-                                    <include>com.google.guava:failureaccess</include>
+                                    <include>*:*</include>
                                 </includes>
+                                <excludes>
+                                    <!-- Exclude unnecessary dependencies.
```

Comment:
Do we need to update the notice file?

Reply:
Already done, this was a no-op change from https://issues.apache.org/jira/browse/FLINK-30089